dotnet/src/SemanticKernel.Core/PromptTemplate/KernelPromptTemplate.cs

```diff
@@ -28,8 +30,9 @@ internal sealed class KernelPromptTemplate : IPromptTemplate
     /// Constructor for PromptTemplate.
     /// </summary>
     /// <param name="promptConfig">Prompt template configuration</param>
+    /// <param name="allowUnsafeContent">Flag indicating whether to allow unsafe content</param>
     /// <param name="loggerFactory">Logger factory</param>
-    public KernelPromptTemplate(PromptTemplateConfig promptConfig, ILoggerFactory? loggerFactory = null)
+    internal KernelPromptTemplate(PromptTemplateConfig promptConfig, bool allowUnsafeContent, ILoggerFactory? loggerFactory = null)
```

Comment:
This is a breaking change. 
Should we keep the public and obsolete it before removing from the public API?

Add the current as a new...

Reply:
The class is internal so the constructor doesn't need to be public.